docs/cli/customer-erasure.md

```diff
@@ -0,0 +1,93 @@
+# Customer Erasure
+
+A package which deletes or exports [commercetools customer data](#list-of-customer-data) in `JSON` format from the [commercetools platform](https://docs.commercetools.com/).
```

Comment:
This is a bit weird: you can either delete or export (so no deletion) customer data. Why is the package called "customer erasure"?

Reply:
Not sure what other names would be good. I just tried to avoid GDPR. What about `Customer Aggregator` or `Customer Collator`. I am find with `Customer Erasure ` as I think it's the package's main purpose and CLI tools are inherently hard to name. The fact that data needs to be gathered in order to be deleted and deletion is an option (as in it can only be exported) seems obvious.

Reply:
Ahh, well said, totally agree.

Reply:
Hmm why not simply `customer-data-exporter`? And you have the option to delete it

Reply:
I don't have a super strong opinion on this but I think using `Erasure` should be fine as it is the main feature clients have been asking for and why we decided to create this package in the first place. 

Even though the deletion part is, indeed, an option in the CLI just to make it a bit safer.

Reply:
I guess `Customer` isn't a perfect match. It is about personal data, which is partially stored in the `Customer` object, but also in `Order` etc.
I'd prefer `Personal Data Foobar` over `Customer Foobar`.

I also agree we shouldn't hardcode GDPR into the name. There may be other countries outside the EU/GDPR that have similar rules.

The law-ish terms are `Retrieval` and `Erasure`, so if we want someone to be able to pattern-match from GDPR to this tool, something like `Personal Data Retrieval and Erasure` would be a good name.

But I don't have a strong opinion, whatever fits well among other CLI tools is also fine with me :)

Reply:
`personal-data-erasure` would be fine by me. I still think erasure is the main purpose of this and the retrieval being a side effect. If someone searches for "how can I delete my private stuff" I don't think `personal-data-exporter` is an obvious name for it. I don't like the term `personal-data-management` as everything is management in software, as is data. 

Long discussions around naming again:

1. Either keep as is
2. Rename to (unless other suggestions): `personal-data-erasure`

Feel free to suggest other names.

Reply:
Yeah, better like this 👌